### PR TITLE
Add tests for FindClassCommandParser and FindClassCommand

### PR DESCRIPTION
**Pull request by yourally2**
- Changed regex so that it throws an error when invalid timing

### CODE REVIEW

**Comment by yourally2** [comment]
I'm not sure why it's causing your tests to fail @s7u4rt99, can you help me look into this? Thanks!

**Comment by s7u4rt99** [comment]
> I'm not sure why it's causing your tests to fail @s7u4rt99, can you help me look into this? Thanks!

Done

File: src/test/java/seedu/address/logic/parser/classcommands/FindClassCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    public void parse_invalidMultiDayArgs_throwsParseException() {
+        assertParseFailure(parser, "FRI SAN",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindClassCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_validMultiDayArgs_returnsFindClassCommand() {
+        FindClassCommand expectedFindClassCommand =
+                new FindClassCommand(new ClassTimingContainsKeywordsPredicate(Arrays.asList("FRI", "SAT")));
+        assertParseSuccess(parser, "FRI SAT", expectedFindClassCommand);
```
Comment (s7u4rt99) [inline src/test/java/seedu/address/logic/parser/classcommands/FindClassCommandParserTest.java:48]: If this parser accepts multiple keywords, how come it doesnt find the classes with fri and sat for the command? The behaviour is not consistent with the findname command

Comment (s7u4rt99) [reply src/test/java/seedu/address/logic/parser/classcommands/FindClassCommandParserTest.java:48]: Other than this, LGTM

Comment (yourally2) [reply src/test/java/seedu/address/logic/parser/classcommands/FindClassCommandParserTest.java:48]: I guess we have to make it such that it only accepts `FRI 0800-1000` in the UG

**Comment by yourally2** [comment]
I can't replicate the error in the checks, I'll just merge it in first

**Comment by s7u4rt99** [comment]
> I can't replicate the error in the checks, I'll just merge it in first

I found the error, it was because u changed the naming of the JC_PHYSICS variable and did not change the import name for the edit class. Thus it was importing a wrong variable. Did u run the checkstyle_test? When i ran it, it showed me that error.

**Pull request merged**